tsconfig.app.json

```
@@ -26,7 +26,17 @@
     /* Types */
     "typeRoots": ["./types", "./node_modules/@types", "./node_modules"],
     "types": ["vitest/globals", "@testing-library/jest-dom"],
-    "rootDirs": [".", "./.react-router/types"]
+    "rootDirs": [".", "./.react-router/types"],
+
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["./src/*"],
+      "@app/*": ["./src/app/*"],
+      "@pages/*": ["./src/pages/*"],
+      "@features/*": ["./src/features/*"],
+      "@entities/*": ["./src/entities/*"],
+      "@shared/*": ["./src/shared/*"]
```

p4
`"@/*": ["./src/*"],`
처럼 `@`하고 `/`를 붙였으면 밑에서도 통일시키는게 좋지않을까요?!

```suggestion
      "@/*": ["./src/*"],
      "@/app/*": ["./src/app/*"],
      "@/pages/*": ["./src/pages/*"],
      "@/features/*": ["./src/features/*"],
      "@/entities/*": ["./src/entities/*"],
      "@/shared/*": ["./src/shared/*"]
```